app/views/email_confirmation.en.html.erb

```diff
@@ -0,0 +1,84 @@
+<html>
```

Comment:
please don't commit template in view folder
use a rails generator and make sure you have the tests:
```

$ rails generate mailer UserMailer
create  app/mailers/user_mailer.rb
create  app/mailers/application_mailer.rb
invoke  erb
create    app/views/user_mailer
create    app/views/layouts/mailer.text.erb
create    app/views/layouts/mailer.html.erb
invoke  test_unit
create    test/mailers/user_mailer_test.rb
create    test/mailers/previews/user_mailer_preview.rb
```

Reply:
Fixed.